core/src/main/java/org/mule/runtime/core/internal/lifecycle/phases/MuleContextStopPhase.java

```diff
@@ -48,7 +49,8 @@ public MuleContextStopPhase() {
         InterceptingMessageProcessor.class,
         Component.class,
         OutboundRouter.class,
-        Service.class
+        Service.class,
+        GlobalErrorHandler.class
```

Comment:
this is added because the stop phase in the global error handler was called when stopping the mule context, but it wasn't called when starting it (the same happened with the dispose)